assignments/array-methods.js

```diff
@@ -94,8 +94,28 @@ console.log('total: ', ticketPriceTotal);
 // ==== Challenge 5: Be Creative ====
 // Now that you have used .forEach(), .map(), .filter(), and .reduce().  I want you to think of potential problems you could solve given the data set and the 5k fun run theme.  Try to create and then solve 3 unique problems using one or many of the array methods listed above.
 
-// Problem 1
+// Problem 1: We want runners' first and last names converted to uppercase.
```

Comment:
Arrays look fantastic! I especially love how creative you got with challenge 5 and trying new methods.